src/main/java/frc/robot/subsystems/DrivetrainSubsystem.java

```diff
@@ -54,6 +54,10 @@ public DrivetrainSubsystem() {
 
     //Created differential drive by using left motors and right motors.
     diffDrive = new DifferentialDrive(leftMotorGroup, rightMotorGroup);
+    leftTopMotor = new WPI_TalonFX(Constants.DrivetrainConstants.LEFT_TOP_MOTOR);
```

Comment:
Oops, you don't want the second copy of these lines to be re-copied back in!  (Have you merged the main branch into this branch recently?)